src/hooks/useFormatNumber.mdx

```diff
@@ -4,43 +4,43 @@ import { Meta, Source } from "@storybook/blocks";
 
 # useFormatNumber
 
-| Hook Version | DS Version |
-| ------------ | ---------- |
+| Hook Version | DS Version   |
+| ------------ | ------------ |
 | Added        | `Prerelease` |
 | Latest       | `Prerelease` |
 
 `useFormatNumber` provides a number with commas and handles a range of numbers.
+The formatting is based on standards from the NYPL Writing Style Guide.
+
+## Usage
```

Comment:
After this heading, add this text:
```
The hook returns an object with the `formatNumber` utility function.
```

And then add this code example:
```
const {
  formatNumber,
} = useFormatNumber();
```

Reply:
Ah, this is not necessarily true because it doesn't work like other hooks. The hook itself is the function that should be used and it returns the string value.

Reply:
@EdwinGuzman Got it. I was aiming to align these docs with the docs for the other hooks where examples like this are shown. What would you suggest?

Reply:
Use the code snippet in the Text example here.